scripts/capsules/capsule_update_tests.sh

```diff
@@ -151,3 +151,57 @@ BaseTools/BinWrappers/PosixLike/GenerateCapsule --encode \
                                                 --capflag PersistAcrossReset \
                                                 --json-file $output_file \
                                                 --output $capsule_name$file_descriptor.cap
+
+
+
+
+
+
+
+
+echo "--- CREATING CAPSULE WITH SERIAL REDIRECTION ---"
```

Comment:
Why is it required? System settings are supposed to be preserved, if they aren't it is an issue with the capsule update mechanism, which should result in test failure.

Capsule created by this fragment isn't used anywhere, please remove changes to this file.

Reply:
You are right. I have added it by mistake - its just my script to edit the capsule so that it contains a serial redirection 'fix' which I use just for myself. Removed in: 36dca57910b698fa7cb95dfe573d9259dea31e2f